portfolio/src/main/webapp/youtube.html

```diff
@@ -4,14 +4,15 @@
     <meta charset="UTF-8">
     <title>Youtube</title>
     <link rel="stylesheet" href="youtube-style.css">
-    <script src="youtube_script.js"></script>
+    <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="loadComments()">
       <form action="/data" method="POST">
         <label for="user-input">What's a youtube video idea you have for me?</label><br>
         <textarea name="user-input">Put your idea here!</textarea>
```

Comment:
make this "required"

Reply:
i also have this in my post-practice branch! merging that branch now!